src/figuresClasses.ts

```diff
@@ -1,19 +1,100 @@
 export interface Figure {
-
+  a?: number,
+  b?: number,
+  c?: number,
+  radius?: number,
+  width?: number,
+  height?: number,
+  getArea(): number,
+  color: 'red' | 'green' | 'blue',
 }
 
 export class Triangle implements Figure {
+  constructor(
```

Comment:
set `shape` for other classes
```suggestion
  shape: Shape = 'triangle';
  
  constructor(
```